src/helm/proxy/clients/auto_client.py

```diff
@@ -219,3 +222,15 @@ def get_toxicity_classifier_client(self) -> PerspectiveAPIClient:
         """Get the toxicity classifier client. We currently only support Perspective API."""
         cache_config: CacheConfig = self._build_cache_config("perspectiveapi")
         return PerspectiveAPIClient(self.credentials.get("perspectiveApiKey", ""), cache_config)
+
+    def get_critique_client(self) -> CritiqueClient:
+        """Get the critique client."""
+        if not self.critique_client:
+            surgeai_credentials = self.credentials.get("surgeaiApiKey", None)
```

Comment:
If we have GPT-4 and mturk, how do we select the right version?

Reply:
This isn't supported yet but I made a note to making client switching configurable in #1453